js/build.js

```diff
@@ -2330,9 +2363,9 @@ Fliplet.Widget.instance('chat', function (data) {
       }
     });
 
-    if (messagesQueue.length) {
-      setTimeout(function() {
-        messagesQueue.forEach(function(message) {
+    if (queue.getAllQueue().length) {
```

Comment:
I would rename `getAllQueue()` to `getMessages()`

Reply:
@squallstar Fixed